[User experience] Fix JS error rate

Kerry350 opened the pull request:
## Summary

This PR **partially** fixes #78926. It incorporates feedback from https://github.com/elastic/kibana/issues/78926#issuecomment-713605265 onwards.

Namely:

- Make sure `exists: { field: 'transaction.marks.navigationTiming.fetchStart' }` doesn't exclude documents in the `getClientMetrics` function queries (so they can contribute towards the page views value).

- Derive `page views` from the hit count of the outer query, rather than using a `value_count` aggregation (or `cardinality` on `transaction.id`).

## Note

The `getRumPageLoadTransactionsProjection` projection is used in a lot of places, so I've added a `checkFetchStartFieldExists` param which is by default set to `true`. I didn't want to add a separate projection just for the `getClientMetrics` query. This can be changed, but was the easiest way forward. There's a comment alongside the `exists` that states `Adding this filter to cater for some inconsistent rum data not available on aggregated transactions` but it's not clear what that RUM data is, and so this should be applied to the smallest piece possible.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/)
* Commit: 2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/execution/node/364/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/","number":"83418","commit":"2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/)
* Commit: 4e9479da97f7cdec451012229da2b3647695114b

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #83418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/) failed 2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/","number":"83418","commit":"2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/","number":"83604","commit":"4e9479da97f7cdec451012229da2b3647695114b"}]}
PIPELINE-->

shahzad31 commented on x-pack/plugins/apm/server/lib/rum_client/get_client_metrics.ts:
```diff
@@ -25,17 +24,14 @@ export async function getClientMetrics({
   const projection = getRumPageLoadTransactionsProjection({
     setup,
     urlQuery,
+    checkFetchStartFieldExists: false,
```
So this should be only excluded for page views, for other aggs like frontend/backend time, we continue to want to use this.

shahzad31 replied:
also we need to exclude this filter for page views chart.

elasticmachine commented:
Pinging @elastic/apm-ui (Team:apm)

elasticmachine commented:
Pinging @elastic/uptime (Team:uptime)

Kerry350 commented:
@shahzad31 This is ready for another look. Addressed your feedback and added the tweaks from https://github.com/elastic/kibana/issues/78926#issuecomment-716696177.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/)
* Commit: 66aa3a071683af11b2b9f401d3765c4782476307
* This comment will update when the build is complete

### Failed CI Steps
* [Check i18n](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/execution/node/384/log)

### History
* :green_heart: [Build #83604](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/) succeeded 4e9479da97f7cdec451012229da2b3647695114b
* :broken_heart: [Build #83418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/) failed 2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/","number":"83418","commit":"2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/","number":"83604","commit":"4e9479da97f7cdec451012229da2b3647695114b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/","number":"83959","commit":"66aa3a071683af11b2b9f401d3765c4782476307"}]}
PIPELINE-->

shahzad31 reviewed (approved):
LGTM,

Waiting for e2e test !!

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84007/)
* Commit: a0776866f78c295ce53ee6ddaaafc43f14305b2f

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### async chunks size
| id | [before](https://github.com/elastic/kibana/commit/441890ccb36efa6626bd64d6bb74f9f37013e81f) | [after](https://github.com/elastic/kibana/commit/a0776866f78c295ce53ee6ddaaafc43f14305b2f) | diff |
| --- | --- | --- | --- |
| `apm` | 3.3MB | 3.3MB | -719.0B |

### History
* :broken_heart: [Build #83959](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/) failed 66aa3a071683af11b2b9f401d3765c4782476307
* :green_heart: [Build #83604](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/) succeeded 4e9479da97f7cdec451012229da2b3647695114b
* :broken_heart: [Build #83418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/) failed 2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/","number":"83418","commit":"2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/","number":"83604","commit":"4e9479da97f7cdec451012229da2b3647695114b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/","number":"83959","commit":"66aa3a071683af11b2b9f401d3765c4782476307"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84007/","number":"84007","commit":"a0776866f78c295ce53ee6ddaaafc43f14305b2f"}]}
PIPELINE-->

Kerry350 commented:
Hrm, functional test failure is unrelated 😞

```
apis
     │       Maps endpoints
     │         
     │           EMS proxy
     │             should correctly rewrite url and format:
     │
     │      Error: expected 'topojson' to equal 'geojson'
```

Kerry350 commented:
@elasticmachine merge upstream

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/)
* Commit: 3738bee726969317ac28e4dc1ac7d41e996d6af2
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/execution/node/627/log)
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/execution/node/727/log)

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/testReport)

<details><summary>X-Pack API Integration Tests.x-pack/test/api_integration/apis/maps/proxy_api·js.apis Maps endpoints  EMS proxy should correctly rewrite url and format</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/testReport//junit/X-Pack%20API%20Integration%20Tests/x-pack_test_api_integration_apis_maps_proxy_api·js/apis_Maps_endpoints__EMS_proxy_should_correctly_rewrite_url_and_format)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 1 times on tracked branches: https://dryrun

[00:00:00]       │
[00:00:00]         │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603818995540702230] [ilm-history-4-000001] creating index, cause [api], templates [ilm-history], shards [1]/[0]
[00:00:00]         └-: apis
[00:00:00]           └-> "before all" hook
[00:05:27]           └-: Maps endpoints
[00:05:27]             └-> "before all" hook
[00:05:27]             └-> "before all" hook
[00:05:27]               │ info [logstash_functional] Loading "mappings.json"
[00:05:27]               │ info [logstash_functional] Loading "data.json.gz"
[00:05:27]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603818995540702230] [logstash-2015.09.22] creating index, cause [api], templates [], shards [1]/[0]
[00:05:27]               │ info [logstash_functional] Created index "logstash-2015.09.22"
[00:05:27]               │ debg [logstash_functional] "logstash-2015.09.22" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:27]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603818995540702230] [logstash-2015.09.20] creating index, cause [api], templates [], shards [1]/[0]
[00:05:27]               │ info [logstash_functional] Created index "logstash-2015.09.20"
[00:05:27]               │ debg [logstash_functional] "logstash-2015.09.20" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:27]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603818995540702230] [logstash-2015.09.21] creating index, cause [api], templates [], shards [1]/[0]
[00:05:27]               │ info [logstash_functional] Created index "logstash-2015.09.21"
[00:05:27]               │ debg [logstash_functional] "logstash-2015.09.21" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:37]               │ info progress: 12247
[00:05:38]               │ info [logstash_functional] Indexed 4634 docs into "logstash-2015.09.22"
[00:05:38]               │ info [logstash_functional] Indexed 4757 docs into "logstash-2015.09.20"
[00:05:38]               │ info [logstash_functional] Indexed 4614 docs into "logstash-2015.09.21"
[00:05:39]             └-: 
[00:05:39]               └-> "before all" hook
[00:05:39]               └-: EMS proxy
[00:05:39]                 └-> "before all" hook
[00:05:39]                 └-> should correctly rewrite url and format
[00:05:39]                   └-> "before each" hook: global before each
[00:05:39]                   └- ✖ fail: apis Maps endpoints  EMS proxy should correctly rewrite url and format
[00:05:39]                   │      Error: expected 'topojson' to equal 'geojson'
[00:05:39]                   │       at Assertion.assert (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:100:11)
[00:05:39]                   │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:227:8)
[00:05:39]                   │       at Assertion.be (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:69:22)
[00:05:39]                   │       at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
[00:05:39]                   │ 
[00:05:39]                   │ 

```

#### Stack Trace
```
Error: expected 'topojson' to equal 'geojson'
    at Assertion.assert (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
```
</details>

---

<details><summary>X-Pack API Integration Tests.x-pack/test/api_integration/apis/maps/proxy_api·js.apis Maps endpoints  EMS proxy should correctly rewrite url and format</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/testReport//junit/X-Pack%20API%20Integration%20Tests/x-pack_test_api_integration_apis_maps_proxy_api·js/apis_Maps_endpoints__EMS_proxy_should_correctly_rewrite_url_and_format_2)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches

[00:00:00]       │
[00:00:00]         └-: apis
[00:00:00]           └-> "before all" hook
[00:05:27]           └-: Maps endpoints
[00:05:27]             └-> "before all" hook
[00:05:27]             └-> "before all" hook
[00:05:27]               │ info [logstash_functional] Loading "mappings.json"
[00:05:27]               │ info [logstash_functional] Loading "data.json.gz"
[00:05:27]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603818995540702230] [logstash-2015.09.22] creating index, cause [api], templates [], shards [1]/[0]
[00:05:27]               │ info [logstash_functional] Created index "logstash-2015.09.22"
[00:05:27]               │ debg [logstash_functional] "logstash-2015.09.22" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:27]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603818995540702230] [logstash-2015.09.20] creating index, cause [api], templates [], shards [1]/[0]
[00:05:27]               │ info [logstash_functional] Created index "logstash-2015.09.20"
[00:05:27]               │ debg [logstash_functional] "logstash-2015.09.20" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:27]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603818995540702230] [logstash-2015.09.21] creating index, cause [api], templates [], shards [1]/[0]
[00:05:27]               │ info [logstash_functional] Created index "logstash-2015.09.21"
[00:05:27]               │ debg [logstash_functional] "logstash-2015.09.21" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:37]               │ info progress: 12252
[00:05:38]               │ info [logstash_functional] Indexed 4634 docs into "logstash-2015.09.22"
[00:05:38]               │ info [logstash_functional] Indexed 4757 docs into "logstash-2015.09.20"
[00:05:38]               │ info [logstash_functional] Indexed 4614 docs into "logstash-2015.09.21"
[00:05:39]             └-: 
[00:05:39]               └-> "before all" hook
[00:05:40]               └-: EMS proxy
[00:05:40]                 └-> "before all" hook
[00:05:40]                 └-> should correctly rewrite url and format
[00:05:40]                   └-> "before each" hook: global before each
[00:05:40]                   └- ✖ fail: apis Maps endpoints  EMS proxy should correctly rewrite url and format
[00:05:40]                   │      Error: expected 'topojson' to equal 'geojson'
[00:05:40]                   │       at Assertion.assert (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:100:11)
[00:05:40]                   │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:227:8)
[00:05:40]                   │       at Assertion.be (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:69:22)
[00:05:40]                   │       at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
[00:05:40]                   │ 
[00:05:40]                   │ 

```

#### Stack Trace
```
Error: expected 'topojson' to equal 'geojson'
    at Assertion.assert (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/7/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
```
</details>

---

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### async chunks size
| id | [before](https://github.com/elastic/kibana/commit/441890ccb36efa6626bd64d6bb74f9f37013e81f) | [after](https://github.com/elastic/kibana/commit/3738bee726969317ac28e4dc1ac7d41e996d6af2) | diff |
| --- | --- | --- | --- |
| `apm` | 3.3MB | 3.3MB | -719.0B |

### History
* :green_heart: [Build #84007](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84007/) succeeded a0776866f78c295ce53ee6ddaaafc43f14305b2f
* :broken_heart: [Build #83959](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/) failed 66aa3a071683af11b2b9f401d3765c4782476307
* :green_heart: [Build #83604](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/) succeeded 4e9479da97f7cdec451012229da2b3647695114b
* :broken_heart: [Build #83418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/) failed 2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/","number":"83418","commit":"2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/","number":"83604","commit":"4e9479da97f7cdec451012229da2b3647695114b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/","number":"83959","commit":"66aa3a071683af11b2b9f401d3765c4782476307"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84007/","number":"84007","commit":"a0776866f78c295ce53ee6ddaaafc43f14305b2f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/","number":"84038","commit":"3738bee726969317ac28e4dc1ac7d41e996d6af2"}]}
PIPELINE-->

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/)
* Commit: ab5baffc012b667c752d8ff0ae8930ba06e7b316
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/execution/node/627/log)
* [Execute xpack-kibana-ciGroup6](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/execution/node/729/log)

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/testReport)

<details><summary>X-Pack API Integration Tests.x-pack/test/api_integration/apis/maps/proxy_api·js.apis Maps endpoints  EMS proxy should correctly rewrite url and format</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/testReport//junit/X-Pack%20API%20Integration%20Tests/x-pack_test_api_integration_apis_maps_proxy_api·js/apis_Maps_endpoints__EMS_proxy_should_correctly_rewrite_url_and_format)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 4 times on tracked branches: https://github.com/elastic/kibana/issues/81844

[00:00:00]       │
[00:00:00]         └-: apis
[00:00:00]           └-> "before all" hook
[00:05:25]           └-: Maps endpoints
[00:05:25]             └-> "before all" hook
[00:05:25]             └-> "before all" hook
[00:05:25]               │ info [logstash_functional] Loading "mappings.json"
[00:05:25]               │ info [logstash_functional] Loading "data.json.gz"
[00:05:25]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603824139841008738] [logstash-2015.09.22] creating index, cause [api], templates [], shards [1]/[0]
[00:05:25]               │ info [logstash_functional] Created index "logstash-2015.09.22"
[00:05:25]               │ debg [logstash_functional] "logstash-2015.09.22" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:25]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603824139841008738] [logstash-2015.09.20] creating index, cause [api], templates [], shards [1]/[0]
[00:05:25]               │ info [logstash_functional] Created index "logstash-2015.09.20"
[00:05:25]               │ debg [logstash_functional] "logstash-2015.09.20" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:25]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603824139841008738] [logstash-2015.09.21] creating index, cause [api], templates [], shards [1]/[0]
[00:05:25]               │ info [logstash_functional] Created index "logstash-2015.09.21"
[00:05:25]               │ debg [logstash_functional] "logstash-2015.09.21" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:35]               │ info progress: 12942
[00:05:36]               │ info [logstash_functional] Indexed 4634 docs into "logstash-2015.09.22"
[00:05:36]               │ info [logstash_functional] Indexed 4757 docs into "logstash-2015.09.20"
[00:05:36]               │ info [logstash_functional] Indexed 4614 docs into "logstash-2015.09.21"
[00:05:36]             └-: 
[00:05:36]               └-> "before all" hook
[00:05:37]               └-: EMS proxy
[00:05:37]                 └-> "before all" hook
[00:05:37]                 └-> should correctly rewrite url and format
[00:05:37]                   └-> "before each" hook: global before each
[00:05:37]                   └- ✖ fail: apis Maps endpoints  EMS proxy should correctly rewrite url and format
[00:05:37]                   │      Error: expected 'topojson' to equal 'geojson'
[00:05:37]                   │       at Assertion.assert (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:100:11)
[00:05:37]                   │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:227:8)
[00:05:37]                   │       at Assertion.be (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:69:22)
[00:05:37]                   │       at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
[00:05:37]                   │ 
[00:05:37]                   │ 

```

#### Stack Trace
```
Error: expected 'topojson' to equal 'geojson'
    at Assertion.assert (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
```
</details>

---

<details><summary>X-Pack API Integration Tests.x-pack/test/api_integration/apis/maps/proxy_api·js.apis Maps endpoints  EMS proxy should correctly rewrite url and format</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/testReport//junit/X-Pack%20API%20Integration%20Tests/x-pack_test_api_integration_apis_maps_proxy_api·js/apis_Maps_endpoints__EMS_proxy_should_correctly_rewrite_url_and_format_2)


#### Standard Out
```
Failed Tests Reporter:
  - Test has failed 4 times on tracked branches: https://github.com/elastic/kibana/issues/81844

[00:00:00]       │
[00:00:00]         └-: apis
[00:00:00]           └-> "before all" hook
[00:05:23]           └-: Maps endpoints
[00:05:23]             └-> "before all" hook
[00:05:23]             └-> "before all" hook
[00:05:23]               │ info [logstash_functional] Loading "mappings.json"
[00:05:23]               │ info [logstash_functional] Loading "data.json.gz"
[00:05:23]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603824139841008738] [logstash-2015.09.22] creating index, cause [api], templates [], shards [1]/[0]
[00:05:23]               │ info [logstash_functional] Created index "logstash-2015.09.22"
[00:05:23]               │ debg [logstash_functional] "logstash-2015.09.22" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:23]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603824139841008738] [logstash-2015.09.20] creating index, cause [api], templates [], shards [1]/[0]
[00:05:23]               │ info [logstash_functional] Created index "logstash-2015.09.20"
[00:05:23]               │ debg [logstash_functional] "logstash-2015.09.20" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:23]               │ info [o.e.c.m.MetadataCreateIndexService] [kibana-ci-immutable-ubuntu-18-tests-xxl-1603824139841008738] [logstash-2015.09.21] creating index, cause [api], templates [], shards [1]/[0]
[00:05:23]               │ info [logstash_functional] Created index "logstash-2015.09.21"
[00:05:23]               │ debg [logstash_functional] "logstash-2015.09.21" settings {"index":{"analysis":{"analyzer":{"url":{"max_token_length":"1000","tokenizer":"uax_url_email","type":"standard"}}},"number_of_replicas":"0","number_of_shards":"1"}}
[00:05:33]               │ info progress: 13242
[00:05:33]               │ info [logstash_functional] Indexed 4634 docs into "logstash-2015.09.22"
[00:05:33]               │ info [logstash_functional] Indexed 4757 docs into "logstash-2015.09.20"
[00:05:33]               │ info [logstash_functional] Indexed 4614 docs into "logstash-2015.09.21"
[00:05:34]             └-: 
[00:05:34]               └-> "before all" hook
[00:05:35]               └-: EMS proxy
[00:05:35]                 └-> "before all" hook
[00:05:35]                 └-> should correctly rewrite url and format
[00:05:35]                   └-> "before each" hook: global before each
[00:05:36]                   └- ✖ fail: apis Maps endpoints  EMS proxy should correctly rewrite url and format
[00:05:36]                   │      Error: expected 'topojson' to equal 'geojson'
[00:05:36]                   │       at Assertion.assert (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:100:11)
[00:05:36]                   │       at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:227:8)
[00:05:36]                   │       at Assertion.be (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:69:22)
[00:05:36]                   │       at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
[00:05:36]                   │ 
[00:05:36]                   │ 

```

#### Stack Trace
```
Error: expected 'topojson' to equal 'geojson'
    at Assertion.assert (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:100:11)
    at Assertion.be.Assertion.equal (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:227:8)
    at Assertion.be (/dev/shm/workspace/parallel/11/kibana/packages/kbn-expect/expect.js:69:22)
    at Context.it (test/api_integration/apis/maps/proxy_api.js:24:45)
```
</details>

---

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### async chunks size
| id | [before](https://github.com/elastic/kibana/commit/1066602ab7c4d52139c6156601f182da653dd80c) | [after](https://github.com/elastic/kibana/commit/ab5baffc012b667c752d8ff0ae8930ba06e7b316) | diff |
| --- | --- | --- | --- |
| `apm` | 3.3MB | 3.3MB | -719.0B |

### History
* :broken_heart: [Build #84038](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/) failed 3738bee726969317ac28e4dc1ac7d41e996d6af2
* :green_heart: [Build #84007](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84007/) succeeded a0776866f78c295ce53ee6ddaaafc43f14305b2f
* :broken_heart: [Build #83959](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/) failed 66aa3a071683af11b2b9f401d3765c4782476307
* :green_heart: [Build #83604](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/) succeeded 4e9479da97f7cdec451012229da2b3647695114b
* :broken_heart: [Build #83418](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/) failed 2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83418/","number":"83418","commit":"2de8bfba691cd3e5e2b4d3ea6995bdcec952ef20"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83604/","number":"83604","commit":"4e9479da97f7cdec451012229da2b3647695114b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/83959/","number":"83959","commit":"66aa3a071683af11b2b9f401d3765c4782476307"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84007/","number":"84007","commit":"a0776866f78c295ce53ee6ddaaafc43f14305b2f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84038/","number":"84038","commit":"3738bee726969317ac28e4dc1ac7d41e996d6af2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/84077/","number":"84077","commit":"ab5baffc012b667c752d8ff0ae8930ba06e7b316"}]}
PIPELINE-->

Kerry350 commented:
@elasticmachine merge upstream

Flaky maps tests have been skipped upstream, giving this another whirl 🤞
